Agents/FeatureInfoAgent/code/WEB-INF/web.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I believe this file is redundant when you use the @WebServlet annotation in your FeatureInfoAgent class

Reply:
You may be right, I'll make a note to try removing it in future (it's not critical to do so now).